resources/detailsview/main.js

```diff
@@ -55,7 +45,7 @@
             return;
         }
 
-        if (href.startsWith("https:")) {
+        if (href.startsWith("https:") || element.classList.contains("js-page-url")) {
```

Comment:
I was getting issues within VS Code where the `js-page-url` link was opening in a new window vs within the panel.